src/NativeDashLine.ios.js

```diff
@@ -0,0 +1,19 @@
+import React, { Component, PropTypes } from 'react'
+import { View, requireNativeComponent } from 'react-native'
+
+export default class NativeDashLine extends Component {
+  static propTypes = {
+    ...View.propTypes,
+    dashColor: PropTypes.string,
+    dashLineWidth: PropTypes.number,
+    dashLineSpace: PropTypes.number,
+  }
+
+  render() {
+    return (
+      <NativeDashLineComponent {...this.props} />
+    )
+  }
+}
+
+const NativeDashLineComponent = requireNativeComponent('NativeDashLineComponent', null)
```

Comment:
Why we don't pass `NativeDashLine` as second argument in `requireNativeComponent` ?